applications/launchpad/backend/src/commands/host.rs

```diff
@@ -21,12 +21,28 @@
 // USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 //
 
-use std::{convert::TryFrom, path::PathBuf, process::Command, time::Duration};
+use std::{
+    convert::TryFrom,
+    path::PathBuf,
+    process::{Command, Stdio},
+    time::Duration,
+};
 
+use bollard::Docker;
 use log::*;
 use tauri::{api::path::home_dir, AppHandle, Wry};
 
-use crate::commands::AppState;
+use crate::{commands::AppState, docker::DOCKER_INSTANCE};
+
+#[tauri::command]
+pub async fn check_docker(_app: AppHandle<Wry>) -> Result<String, String> {
```

Comment:
```suggestion
pub async fn check_docker() -> Result<Version, String> {
```

Reply:
1. If you don't need `AppHandle`, don't include it
2. Return the object, not a string